apps/passport-server/src/routing/middlewares/tracingMiddleware.ts

```diff
@@ -0,0 +1,33 @@
+import { NextFunction, Request, Response } from "express";
+import { traced } from "../../services/telemetry";
+
+/**
+ * Express middleware that traces all HTTP requests to HoneyComb. This
+ * makes it possible to inspect the performance characteristics of all
+ * HTTP requests.
+ */
+export function tracingMiddleware() {
```

Comment:
This is quite lit that Express makes this so easy to do instead of having to wrap everything. Starting to appreciate Express more